.github/workflows/pr.yml

```diff
@@ -19,9 +19,27 @@ jobs:
   test:
     uses: ./.github/workflows/test.yml
 
+  teams:
+    runs-on: ubuntu-latest
+    if: github.base_ref == 'develop'
+    steps:
+      - uses: convictional/trigger-workflow-and-wait@v1.6.1
+        with:
+          owner: voxel51
+          repo: fiftyone-teams
+          github_token: ${{ secrets.FIFTYONE_GITHUB_TOKEN }}
+          github_user: voxel51-bot
+          workflow_file_name: merge-oss.yml
```

Comment:
Can you link to this file?

Reply:
Yes, the automation can find this in the private repository. Can take offline